gardening/data/documents_folder.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+  <record id="documents_folder_9" model="documents.folder">
+    <field name="name">Internal</field>
+    <field name="parent_folder_id" ref="documents_project.documents_project_folder"/>
+  </record>
+  <record id="documents_folder_12" model="documents.folder">
```

Comment:
Where is it used? Seems like those folders should be in demo

Reply:
in the project_project.xml